### PR TITLE
Add normalize.schemas config to SR client

### PR DESCRIPTION
**Pull request by rayokota**

### CODE REVIEW

File: src/Confluent.SchemaRegistry/ISchemaRegistryClient.cs [hunk 1]
```diff
@@ -42,11 +42,14 @@ public interface ISchemaRegistryClient : IDisposable
         /// <param name="avroSchema">
         ///     The schema to register.
         /// </param>
+        /// <param name="normalize">
+        ///   Whether to normalize schemas.
```
Comment (mhowlett) [inline src/Confluent.SchemaRegistry/ISchemaRegistryClient.cs:46]: nit: 4 spaces indentation

File: test/Confluent.SchemaRegistry.IntegrationTests/Tests/RegisterNormalizedSchema.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                "nt\",\"null\"]},{\"name\":\"favorite_color\",\"type\":[\"string\",\"null\"]}]}";
+
+            var sr = new CachedSchemaRegistryClient(new SchemaRegistryConfig { Url = config.Server });
+
+            var subject = SubjectNameStrategy.Topic.ConstructKeySubjectName(topicName, "Confluent.Kafka.Examples.AvroSpecific.User");
+            Assert.Equal(topicName + "-key", subject);
+
+            var id1 = sr.RegisterSchemaAsync(subject, testSchema1, true).Result;
+            var id2 = sr.GetSchemaIdAsync(subject, testSchema1, true).Result;
+            Assert.Equal(id1, id2);
```
Comment (mhowlett) [inline test/Confluent.SchemaRegistry.IntegrationTests/Tests/RegisterNormalizedSchema.cs:43]: maybe you could also get the schema and check it's different? (add some whitespace in testSchema1 or something)?

**Review by mhowlett — APPROVED** [review]

File: test/Confluent.SchemaRegistry.IntegrationTests/Tests/RegisterNormalizedSchema.cs (hunk 2 above)
Comment (rayokota) [reply test/Confluent.SchemaRegistry.IntegrationTests/Tests/RegisterNormalizedSchema.cs:49]: Done

File: src/Confluent.SchemaRegistry/ISchemaRegistryClient.cs (hunk 1 above)
Comment (rayokota) [reply src/Confluent.SchemaRegistry/ISchemaRegistryClient.cs]: Fixed

**Pull request merged**